kronjob/_kronjob.py

```diff
@@ -216,4 +250,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    print_cronjobs()
```

Comment:
Just want to make sure, this doesnt change the external interface for calling this function correct?

Reply:
it does, the importable name is `print_cronjobs`. I'll check if we're doing it

Reply:
oh shoot, this is a public repo. I'll swap back to `main`.

Reply:
Technically since this is open source, anyone who is using this externally may have it fail as well. Although I agree with you that this makes sense, might have to save this for a more comprehensive update with a major version update.